src/metabase/lib/js.cljs

```diff
@@ -113,6 +113,16 @@
        (update :table update-keys u/->camelCaseEn)
        (clj->js :keyword-fn u/qualified-name))))
 
+(defn ^:export display-name
+  "Given an opaque Cljs object, return a plain JS object with a name useful for display.
+  See `:metabase.lib.metadata.calculation/display-name` ."
+  ([a-query x]
+   (display-name a-query -1 x))
+  ([a-query stage-number x]
+   (display-name a-query stage-number x "default"))
+  ([a-query stage-number x style]
+   (lib.core/display-name a-query stage-number x (keyword style))))
```

Comment:
Isn't this missing the conversion of the result to JS stuff?

Reply:
Since it's returning a string its (iiuc) unnecessary. But I remove since it's unwanted.